VideoLocker/res/layout/create_new_item_layout.xml

```diff
@@ -23,11 +23,12 @@
             android:layout_height="wrap_content"
             android:layout_toEndOf="@+id/create_new_icon_view"
             android:layout_toRightOf="@+id/create_new_icon_view"
-            android:layout_marginStart="10dp"
-            android:layout_marginLeft="10dp"
+            android:layout_marginStart="3dp"
+            android:layout_marginLeft="3dp"
             android:layout_centerVertical="true"
             android:textColor="@color/white"
```

Comment:
This should probably be `@color/edx_grayscale_neutral_white`, as that's what's defined in the `android:textColor` item in the `footer_action_button` style (which can be removed, as it's being applied to the parent). Instead, you should probably create a text style for this color, and set it as the value of the `textAppearance` here.